20_aws_networking/aws_alb.tf

```diff
@@ -0,0 +1,59 @@
+resource "aws_lb" "public" {
```

Comment:
Naming should be more precise not just public. If you will need second public how you will name it?